CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
```

Comment:
Should we add some statement of policy about what kind of contributions are desired?

Reply:
Yeah! I like that idea if there are explicit thing (e.g., feature parity with the API or something?)

Reply:
I'll omit this for now since I'd like to get this merged because I know there are internal contributions coming soon and I don't want perfecting this section to block having these docs merged.